mediator-coordination/src/web/handler.rs

```diff
@@ -0,0 +1,333 @@
+use axum::{
+    extract::State,
+    response::{IntoResponse, Response},
+    Extension, Json,
+};
+use didcomm::Message;
+use hyper::{header::CONTENT_TYPE, StatusCode};
+use std::sync::Arc;
+
+use crate::{
+    constant::{DIDCOMM_ENCRYPTED_MIME_TYPE, KEYLIST_UPDATE_2_0},
+    web::{self, error::MediationError, AppState},
+};
+
+#[axum::debug_handler]
+pub async fn process_didcomm_message(
+    State(state): State<Arc<AppState>>,
+    Extension(message): Extension<Message>,
+) -> Response {
+    let delegate_response = match message.type_.as_str() {
```

Comment:
This handler processes all DIDComm messages sent to the server. Make sure that it supports Mediation requests.

Reply:
Looks like the following checks were lost on the unified route:

https://github.com/adorsys/didcomm-mediator-rs/blob/b8fd01ebe64ea0d4da853ceb4046b5dff0bde4dc/mediator-coordination/src/web/coord/handler/mod.rs#L34-L40

Add them to `stateless::process_plain_mediation_request_over_dics`, so it is not lost in the future. When done, I believe function `process_didcomm_mediation_request_message` could be safely removed.

https://github.com/adorsys/didcomm-mediator-rs/blob/b8fd01ebe64ea0d4da853ceb4046b5dff0bde4dc/mediator-coordination/src/web/coord/handler/mod.rs#L45-L56

Reply:
Does it means the case for the stateful feature is not more handle ?

Reply:
You're now handling that path on the unified route. The logic was duplicated up till now.

Reply:
Hum, a little bit confused here